specification/README.md

```diff
@@ -0,0 +1,65 @@
+# Docker App Specification
```

Comment:
Let's use _Docker Application Package_ when referring to collecting of files/image.

Reply:
👍